Integrations/illuminate/illuminate.py

```diff
@@ -0,0 +1,313 @@
+
+    return enrichment_data
+
+
+def ip_command(client: Client, args: dict) -> EnrichmentOutput:
+    ip: str = str(args.get('ip'))
+    enrichment_data: EnrichmentOutput = client.enrich_indicator(ip, 'ip')
+
+    if enrichment_data.has_context_data():
+        enrichment_data.generate_reputation_context('Address', ip, 'ip', 'IP')
+
+    return enrichment_data
+
+
+def file_command(client: Client, args: dict) -> EnrichmentOutput:
+    file: str = str(args.get('file'))
+    enrichment_data: EnrichmentOutput = client.enrich_indicator(file, 'file')
+
+    if enrichment_data.has_context_data():
+        hash_type = get_hash_type(file)
+        if hash_type != 'Unknown':
+            enrichment_data.generate_reputation_context(hash_type.upper(), file, 'file', 'File')
+
+    return enrichment_data
+
+
+def illuminate_enrich_string_command(client: Client, args: dict) -> EnrichmentOutput:
+    string: str = str(args.get('string'))
+    return client.enrich_indicator(string, 'string')
+
+
+def illuminate_enrich_ipv6_command(client: Client, args: dict) -> EnrichmentOutput:
+    ip: str = str(args.get('ip'))
+    return client.enrich_indicator(ip, 'ipv6')
+
+
+def illuminate_enrich_mutex_command(client: Client, args: dict) -> EnrichmentOutput:
+    mutex: str = str(args.get('mutex'))
+    return client.enrich_indicator(mutex, 'mutex')
+
+
+def illuminate_enrich_http_request_command(client: Client, args: dict) -> EnrichmentOutput:
+    http_request: str = str(args.get('http-request'))
+    return client.enrich_indicator(http_request, 'httpRequest')
+
+
+def url_command(client: Client, args: dict) -> EnrichmentOutput:
+    url: str = str(args.get('url'))
+    enrichment_data: EnrichmentOutput = client.enrich_indicator(url, 'url')
+
+    if enrichment_data.has_context_data():
+        enrichment_data.generate_reputation_context('Data', url, 'url', 'URL')
+
+    return enrichment_data
+
+
+''' EXECUTION '''
+
+
+def main():
+    commands = {
+        'domain': domain_command,
+        'email': email_command,
+        'file': file_command,
+        'ip': ip_command,
+        'url': url_command,
+        'illuminate-enrich-string': illuminate_enrich_string_command,
+        'illuminate-enrich-ipv6': illuminate_enrich_ipv6_command,
+        'illuminate-enrich-mutex': illuminate_enrich_mutex_command,
+        'illuminate-enrich-http-request': illuminate_enrich_http_request_command
+    }
+
+    command: str = demisto.command()
+    LOG(f'command is {command}')
+
+    try:
+        client = build_client(demisto.params())
+
+        if command == 'test-module':
+            perform_test_module(client)
+            demisto.results('ok')
+        elif command in commands:
+            enrichment_output: EnrichmentOutput = commands[command](client, demisto.args())
+            enrichment_output.return_outputs()
+    except Exception as e:
+        err_msg = f'Error in {INTEGRATION_NAME} Integration [{e}]'
```

Comment:
please add an import of traceback at the top of the code, and here change the line to 
```suggestion
        err_msg = f'Error in {INTEGRATION_NAME} Integration [{e}]\nTrace:\n{traceback.format_exc()}'
```